web_src/css/review.css

```diff
@@ -61,7 +61,13 @@
 .comment-code-cloud {
   padding: 0.5rem 1rem !important;
   position: relative;
+  margin: 0 auto;
+  max-width: 1000px;
```

Comment:
Why not unlimited?

Reply:
removed unnecessary rules in https://github.com/go-gitea/gitea/pull/24809/commits/4f8e698e8d362d1feacf8b4d37f10564f73c7881, I think `strech` might not needed because the contents already taken the full width and height of parent